From dbd2f1bbce0ab6b235a065de3926a7133f5a77ae Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Mon, 14 Dec 2020 15:39:31 +0200
Subject: [PATCH 29/78] dts: devicetree: Add 'fsl,s32g274a-siul2-gpio' to
 gpio-s32v

'fsl,s32g274a-siul2-gpio' is used for s32g274a to restrict opad ranges

Issue: ALB-6155
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
---
 Documentation/devicetree/bindings/gpio/gpio-s32v.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/gpio/gpio-s32v.txt b/Documentation/devicetree/bindings/gpio/gpio-s32v.txt
index 08fdd1428262..24a3ddcc3573 100644
--- a/Documentation/devicetree/bindings/gpio/gpio-s32v.txt
+++ b/Documentation/devicetree/bindings/gpio/gpio-s32v.txt
@@ -1,7 +1,7 @@
 * Freescale S32V SIUL2 GPIO controller
 
 Required properties:
-- compatible : Should be "fsl,s32v234-siul2-gpio", "fsl,s32gen1-siul2-gpio"
+- compatible : Should be "fsl,s32v234-siul2-gpio", "fsl,s32gen1-siul2-gpio", "fsl,s32g274a-siul2-gpio"
 - reg : 4 or 5 ranges. 4 for nodes without EIRQ management and 5 for others.
 	Ranges index meaning :
 		- 0 Address and length of the Parallel GPIO Pad Data Out Register
-- 
2.25.1

